components/gateway/internal/gqlschema/schema.graphql

```diff
@@ -274,19 +287,26 @@ input FetchRequestInput {
 input ApplicationWebhookInput {
     type: ApplicationWebhookType!
     url: String!
-    credential: CredentialInput
+    auth: AuthInput
 }
 
 # API Input
-
+# you need to perform separate call to set auth
 input APIDefinitionInput {
     targetURL: String!
-    credential: CredentialInput
```

Comment:
If there would be a default `Auth` like I suggested in my previous comment, we could add back optional `auth` field to allow users to specify the default `Auth`.